pipelines/analysis_matched.yaml

```diff
@@ -8,6 +8,22 @@ tasks:
       python: |
         from metric_pipeline_tasks import PA1Task
         config.measure.retarget(PA1Task)
+  PA2:
+    class: metric_pipeline_tasks.MatchedCatalogAnalysisTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PA2
```

Comment:
I've not thought this through completely, but it might be useful to mention which threshold is being used, e.g. `PA2_design` if the default thresholds correspond with design thresholds.

Reply:
Done in the pipeline yaml config.